tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -111,17 +112,17 @@ Feature: antivirus
       | spaces           | eicar_com.zip | virusFile2.zip |
 
 
-  Scenario Outline: upload a file with the virus to a password-protected public share
+  Scenario Outline: public uploads a file with the virus to a password-protected public share
     Given using <dav-path-version> DAV path
+    And using SharingNG
     And user "Alice" has created folder "/uploadFolder"
-    And user "Alice" has created a public link share with settings
-      | path        | /uploadFolder            |
-      | name        | sharedlink               |
-      | permissions | change                   |
-      | password    | newpasswd                |
-      | expireDate  | 2040-01-01T23:59:59+0100 |
-      | password    | %public%                 |
-    When user "Alice" uploads file "filesForUpload/filesWithVirus/<file-name>" to "/<new-file-name>" using the WebDAV API
+    And user "Alice" has created the following link share:
+      | resource           | uploadFolder             |
+      | space              | Personal                 |
+      | permissionsRole    | edit                     |
+      | password           | %public%                 |
+      | expirationDateTime | 2040-01-01T23:59:59.000Z |
+    When the public uploads file "filesForUpload/filesWithVirus/<file-name>" inside last created public link shared folder to "<new-file-name>" with password "%public%" using the new public WebDAV API
```

Comment:
```suggestion
    When the public uploads file "filesForUpload/filesWithVirus/<file-name>" to "<new-file-name>" inside last link shared folder with password "%public%" using the new WebDAV API
```

Reply:
Any reason for the wording to be different? @saw-jan